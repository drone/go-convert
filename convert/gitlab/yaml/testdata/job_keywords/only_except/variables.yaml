# https://docs.gitlab.com/ee/ci/yaml/#onlyvariables--exceptvariables

deploy:
  script: cap staging deploy
  only:
    variables:
      - $RELEASE == "staging"
      - $STAGING