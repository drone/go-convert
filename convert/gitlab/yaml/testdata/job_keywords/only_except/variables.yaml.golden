deploy:
    only:
        variables:
            - $RELEASE == "staging"
            - $STAGING
    script:
        - cap staging deploy
